CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 1.0.0-0
+
+### Breaking Changes
+
+* Run fixture cleanup asynchronously, after teardown
+* Remove callback from beforeEach / afterEach
+* Improve tap-snapshot folder structure
+* Inherit t.saveFixture boolean
+
+### Features
+
+* Create fixture symlinks as junctions if pointing at directories
+* Add tap-testdir- to the generated test dir folder
+* Add `t.mock()` API
+* Add `t.before(fn)` API
+* beforEach / afterEach can return promises
```

Comment:
Typo, also they could always return promises, they just _also_ had a callback they could use if they didn't.  The breaking change is "`beforeEach/afterEach` no longer receive callbacks, and are assumed to be synchronous if they do not return a Promise".